src/test/java/LoginTest.java

```diff
@@ -1,34 +1,190 @@
+            if (returnList.contains(str)) {
+                i--;
+                continue;
+            }
+            returnList.add(str);
+        }
+        return returnList;
+    }
+
+    @Test
+    public void javaScriptConfirmBoxOKTest() {
+        driver.get(Constants.ALERTS_URL);
+        WebElement clickMeButton = driver.findElement(clickMeBtn);
+        clickMeButton.click();
+        driver.switchTo().alert().accept();
+        WebElement resultMessage = driver.findElement(resultMsg);
+        Assert.assertEquals("You pressed OK!", resultMessage.getText());
+    }
+
+    @Test
+    public void javaScriptConfirmBoxCancelTest() {
+        driver.get(Constants.ALERTS_URL);
+        WebElement clickMeButton = driver.findElement(clickMeBtn);
+        clickMeButton.click();
+        driver.switchTo().alert().dismiss();
+        WebElement resultMessage = driver.findElement(resultMsg);
+        Assert.assertEquals("You pressed Cancel!", resultMessage.getText());
+    }
+
+    @Test
+    public void javaScriptAlertBoxTest() {
+        driver.get(Constants.ALERTS_URL);
+        WebElement clickMeButton = driver.findElement(clickAlertBtn);
+        clickMeButton.click();
+        driver.switchTo().alert().sendKeys("Peter");
+        driver.switchTo().alert().accept();
+        WebElement resultMessage = driver.findElement(resultMsgAlert);
+        Assert.assertEquals("You have entered 'Peter' !", resultMessage.getText());
+    }
+
+    @Test
+    public void getRandomUserTest() {
+        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(10));
+        driver.get(Constants.USERS_URL);
+        WebElement clickMeButton = driver.findElement(clickMeRandomBtn);
+        clickMeButton.click();
+        wait.until(ExpectedConditions.visibilityOf(driver.findElement(newUser)));
+        WebElement newParticipant = driver.findElement(newUser);
+        Assert.assertTrue(newParticipant.isDisplayed());
+    }
+
+    @Test
+    public void downloadTest() {
+        driver.get(DOWNLOAD_URL);
+        WebElement clickMeButton = driver.findElement(clickBtnPercent);
+        WebElement percent = driver.findElement(percentage);
+
+        clickMeButton.click();
+        new WebDriverWait(driver, Duration.ofSeconds(15), Duration.ofMillis(100))
+                .until(ExpectedConditions.textToBePresentInElement(percent, "50%"));
+        driver.navigate().refresh();
+    }
+
+    @Test
+    public void downloadTestx() {
+        driver.get(DOWNLOAD2_URL);
+        WebElement dropdown = driver.findElement(By.name("example_length"));
+        Select s = new Select(dropdown);
+        s.selectByValue("10");
+        Assert.assertTrue(!getResultList(60, 400000).isEmpty());
+    }
+
+    public List<Employee> getResultList(int minAge, int maxSalary) {
+        List<Employee> results = new ArrayList<>();
+
+        WebElement nextBtn = driver.findElement(By.id("example_next"));
+        while (!nextBtn.getAttribute("class").contains("disabled")) {
```

Comment:
use the `do while` loop. This will help you avoid repeating 'results.addAll()' multiple times

```
do {
            WebElement nextBtn = driver.findElement(By.id("example_next"));
            results.addAll(getEmployee(minAge, maxSalary));
            nextBtn.click();
        } while (!driver.findElement(By.id("example_next")).getAttribute("class").contains("disabled"));
```